ui/src/components/graph/Topology.vue

```diff
@@ -109,7 +109,7 @@
 
     const editorWidthStorageKey = "editor-width";
     const editorWidthPercentage = ref(localStorage.getItem(editorWidthStorageKey));
-    const isHorizontal = ref(localStorage.getItem("topology-orientation") !== "0");
+    const isHorizontal = ref(localStorage.getItem("topology-orientation") === "1");
```

Comment:
Not right way, you only inverse on the whole application, please pass a props that : 
- force the orientation 
- remove the button